app.py

```diff
@@ -53,6 +53,7 @@ def exportBag():
             manifest_io.status_codes['stood_up_workspace'] = \
                 r_create.status_code
         else:  # something went wrong when creating workspace
+            body, headers = __check_response(r_create)
```

Comment:
Method `__check_response` does not exist.

Ideally a unit test would have caught this, and it would be good to have one.

Reply:
I corrected this. Checking for existence of methods is useful but I need to read more about it to see what's the best approach in unit tests (i.e., did not add a test).

Reply:
Your tests typically wouldn't directly check for the existence of method.

What your test would need to do is execute that piece of code, which would then fail due to the method not being defined. The fact that your tests passed means that your tests are not testing this part of the code.